definitions/features/foreman_database.rb

```diff
@@ -7,6 +7,12 @@ class Features::ForemanDatabase < ForemanMaintain::Feature
     end
   end
 
+  def count(table, condition = '1=1')
+    query("SELECT count(*) AS count FROM #{table} WHERE #{condition}").first['count'].to_i
+  rescue CSV::MalformedCSVError
```

Comment:
would it be good to add logger statement here in rescue block?

Reply:
Yes, makes sense.